### PR TITLE
feat: different Nutri-Score icons and text for unknown and not-applicable 

### LINKED ISSUE #6276: Different icons and text for not applicable Nutri-Score (e.g. alcohol, baby food) in product attributes and knowledge panels
Different icons and text for not applicable Nutri-Score (e.g. alcohol, baby food) in product attributes and knowledge panels### Is your feature request related to a problem? Please describe.

When the Nutri-Score is not applicable for a product category, we currently show the same image and text as when we do not have enough information to compute the Nutri-Score.

- Related: https://github.com/openfoodfacts/openfoodfacts-server/issues/6223

### Describe the solution you'd like

Create specific text and icons for non-applicable Nutri-Score, possibly with a different message based on categories.

### PR DESCRIPTION
**Pull request by stephanegigandet**
Differentiate between Nutri-Score computed because we miss some data (category or nutrition facts) and Nutri-Score not applicable for a category.

Fixes #6276

image

image

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -9040,15 +9038,17 @@ sub data_to_display_nutriscore_and_nutrient_levels($) {
...
 		if (has_tag($product_ref,"misc","en:nutriscore-not-applicable")) {
 				push @nutriscore_warnings, lang("nutriscore_not_applicable");
+				$result_data_ref->{nutriscore_grade} = "not-applicable";
 				$result_data_ref->{nutriscore_unknown_reason} = "not_applicable";
 				$result_data_ref->{nutriscore_unknown_reason_short} = lang("nutriscore_not_applicable_short");
 		}		
 		else {
+
+			$result_data_ref->{nutriscore_grade} = "unknown";
+
```
Comment (alexgarel) [inline lib/ProductOpener/Display.pm:9051]: General bug:
On /nutrition-grades facet page, the graphic does not correspond to the facets that are underneath (the not-applicable does not appears)

Comment (stephanegigandet) [reply lib/ProductOpener/Display.pm:9051]: It is working for me, I see the image:

image

Note that the SVG images need to be rebuilt (optimized) with "make up".

Comment (alexgarel) [reply lib/ProductOpener/Display.pm:9051]: Sorry, I speak about the **chart** above the list.

Comment (stephanegigandet) [reply lib/ProductOpener/Display.pm:9051]: Got it! Fixed.

**Review by alexgarel — APPROVED** [review]
LGTM

**Pull request merged**